boards/others/esp32c3_supermini/esp32c3_rust.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Please rename this file from `esp32c3_rust.yaml` to `esp32c3_supermini.yaml` instead.

Reply:
Done. Don't know how I missed one.